src/test/java/uk/gov/justice/digital/hmpps/keyworker/integration/KeyworkerServiceIntegrationTest.kt

```diff
@@ -20,10 +20,26 @@ class KeyworkerServiceIntegrationTest : IntegrationTest() {
   @Test
   fun beforeEach() {
     migratedFoAutoAllocation(PRISON_ID)
-    eliteMockServer.stubOffendersAtLocationForAutoAllocation(PRISON_ID, OFFENDERS_AT_LOCATION)
+    eliteMockServer.stubOffendersAtLocationForAutoAllocation(OFFENDERS_AT_LOCATION)
     eliteMockServer.stubKeyworkerRoles(PRISON_ID, KEYWORKER_ID_1, STAFF_LOCATION_ROLE_LIST)
   }
 
+  @Test
+  fun `Allocation history for offender reports ok`() {
```

Comment:
found out that was no test for the allocation history for an offender endpoint, needed one in order to load ehcache with some data